### PR TITLE
add invariant project

### PR DESCRIPTION
**Pull request by PawelKrawczyk96**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/invariant exports TVL: 
        
 
 
    
    total                    448.48 k

**Comment by wojciech-cichocki** [comment]
LGTM

File: projects/invariant/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const retry = require("async-retry");
+const axios = require("axios");
+
+async function fetch() {
+  const response = (
+    await retry(
+      async () => await axios.get("https://stats.invariant.app/short/mainnet")
```
Comment (g1nt0ki) [inline projects/invariant/index.js:7]: How difficult is it to fetch the tvl by querying solana rpc instead? I know we dont have the best support for solana, but if you hop onto our discord, I can help with the adapter change if you are struck somewhere

File: pull_request_template.md [hunk 2]
```diff
@@ -1,40 +1,27 @@
-##### Twitter Link:
+##### Twitter Link: https://twitter.com/invariant_labs
```
Comment (g1nt0ki) [inline pull_request_template.md:1]: Please revert the changes to this file

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/invariant/index.js (hunk 1 above)
Comment (wojciech-cichocki) [reply projects/invariant/index.js:7]: Calculating total tvl is quite complicated since we don't store any stats on-chain.
1. Total tvl is the sum of all pools. Currently there are 22 pools, but we anticipate many more as anyone can add their own pool from UI and it only costs a few dollars.
2. Computing tvl for a single pool is also not trivial - it requires fetching all open positions since the program stores "virtual liquidity" due to a concentrated liquidity mechanism.

Overall all functions to compute total tvl are public, but requires fetching all positions and all pools.

Comment (g1nt0ki) [reply projects/invariant/index.js:7]: I understand, can you revert the change to the template file, I would merge this PR

**Comment by llamatester** [comment]
The adapter at projects/invariant exports TVL: 
        
 
 
    
    total                    440.94 k

**Comment by llamatester** [comment]
The adapter at projects/invariant exports TVL: 
        
 
 
    
    total                    440.94 k

**Comment by llamatester** [comment]
The adapter at projects/invariant exports TVL: 
        
 
 
    
    total                    440.94 k

File: pull_request_template.md (hunk 2 above)
Comment (PawelKrawczyk96) [reply pull_request_template.md]: done
